Assets/Scripts/BuffPanelSDK/CookieExtractor.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Reflection;
+using Mono.Data.Sqlite;
+using UnityEngine;
+
+namespace BuffPanel
+{
+    public class CookieExtractor
+    {
+        public static Dictionary<string, string> ReadCookies(string gameToken)
```

Comment:
In general the whole implementation is lacking try, catch blocks, to mitigate any unexpeceted errors. I advise to wrap each call (ie. for each OS/browser/version combination) into an individual block to minimize lost data and to eliminate the risk of a crash